src/isPalindrome.js

```diff
@@ -0,0 +1,11 @@
+export default function isPalidrome (str){
+  var re = /[^A-Za-z0-9]/g;
```

Comment:
If we're using ES6 (`export default`), we should do so consistently - *especially* to avoid potential [hoisting](https://github.com/getify/You-Dont-Know-JS/blob/master/scope%20%26%20closures/ch4.md) bugs by using `let` and `const`.